packages/million/block.ts

```diff
@@ -146,6 +146,11 @@ export class Block extends AbstractBlock {
             insertBefore$.call(el, parent, childNodes[edit.i!]);
             continue;
           }
+
+          if (value === null || value === undefined || value === false) {
```

Comment:
```suggestion
          if (value == null || value === false) {
```

This can be simplified.

Reply:
Use of ```==``` is against ESLint rule (```eqeqeq``` in ```@vercel/style-guide/eslint/node```)